src/plugins/gmx/gmxplugin.cpp

```diff
@@ -110,16 +110,87 @@ bool GmxPlugin::write(const Map *map, const QString &fileName)
     writeProperty(stream, map, "speed", 30);
     writeProperty(stream, map, "persistent", false);
     writeProperty(stream, map, "clearDisplayBuffer", true);
+    writeProperty(stream, map, "clearViewBackground", false);
+    // Check if views are defined
+    bool enableViews = false;
+    LayerIterator iterator(map);
+    while (const Layer *layer = iterator.next()) {
+
+        if (layer->layerType() != Layer::ObjectGroupType)
+            continue;
+
+        const ObjectGroup *objectLayer = static_cast<const ObjectGroup*>(layer);
+
+          for (const MapObject *object : objectLayer->objects()) {
```

Comment:
Coding style: please fix up the indentation (this line is indented too much, and lines below don't use 4 spaces indent)